### PR TITLE
Crain adjust screen size

### PR DESCRIPTION
**Pull request by CSegotaDev**

### CODE REVIEW

File: src/components/sidenav/Tags.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import Button from "@mui/material/Button";
+
+import ModuleMetaData from "../../data/ModuleMetadata";
+
+function Tags() {
+  const tagHolder = [];
+  const frequencyArray = [];
+  let found;
+
+  //take each metadata object and extract the keywords from each.  place keywords in tagHolder array
```
Comment (CSegotaDev) [inline src/components/sidenav/Tags.js:12]: Great code commenting here!

File: src/modules/FilteredLearningModule.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ function FilteredLearningModule(props) {
     <LearningModuleListItem key={metaDatum.id} moduleMetadata={metaDatum} />
   ));
 
+
```
Comment (CSegotaDev) [inline src/modules/FilteredLearningModule.js:13]: I saw this and it reminded me of a time where I decided to change some code formatting in a bunch of places, along with my new code. I adding some spaces, and new lines here and there, things like that. My lead at time time let me know that lots of little changes like that add more files to the PR's, and that they can distract from the actual code changes. -- You only did it once and is really not worth commenting on, but I figured I'd take the opportunity to share the experience.

**Review by CSegotaDev — COMMENTED** [review]
This all looks great! Just a couple comments unrelated to the code itself!

File: src/modules/FilteredLearningModule.js (hunk 2 above)
Comment (CSegotaDev) [reply src/modules/FilteredLearningModule.js:13]: @pcrain9 testing pings.

Comment (pcrain9) [reply src/modules/FilteredLearningModule.js:13]: @CSegotaDev testing!

**Review by pcrain9 — APPROVED** [review]

**Pull request merged**